konduit-serving-api/src/main/java/ai/konduit/serving/util/PortUtils.java

```diff
@@ -0,0 +1,20 @@
+package ai.konduit.serving.util;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+public class PortUtils {
```

Comment:
Copyright header, also add `@NoArgsConstructor`

Reply:
Sorry about that. Keep forgetting about that...